### ISSUE TITLE
Pagination modal is broken on pydantic 1.8

### ISSUE THREAD

**Issue by LeiShi1313** [uriyyo/fastapi-pagination#16]
Sample traceback:
```
Traceback (most recent call last):
   File "app/main.py", line 18, in <module>
     from app.api.v2.servers import servers_v2_router
   File "/app/app/api/v2/servers.py", line 54, in <module>
     response_model=Page[ServerOut],
   File "/usr/local/lib/python3.8/site-packages/pydantic/generics.py", line 85, in __class_getitem__
     create_model(
   File "pydantic/main.py", line 974, in pydantic.main.create_model
   File "pydantic/main.py", line 287, in pydantic.main.ModelMetaclass.__new__
   File "pydantic/fields.py", line 391, in pydantic.fields.ModelField.infer
   File "pydantic/schema.py", line 926, in pydantic.schema.get_annotation_from_field_info
 ValueError: On field "total" the following field constraints are set but not enforced: ge.
 For more details see https://pydantic-docs.helpmanual.io/usage/schema/#unenforced-field-constraints
```
Downgrade to 1.7.3 solve the problem.

**Comment by uriyyo** [comment]
Thanks for submitting the issue, I will try to resolve it as soon as possible.

**Comment by uriyyo** [comment]
Hi @LeiShi1313 this issue is fixed in version `0.5.3`. In case if this issue will appear again, please reopen this issue.

**Comment by LeiShi1313** [comment]
It works, nice! Thank you for fast response!
